README.md

```diff
@@ -12,45 +12,151 @@ Snapfu is the scaffolding command line tool for the Searchspring Snap SDK. This
 npm install -g snapfu
 ```
 
-## Login
+## Usage
 
-Login to access your Github organizations - the following command will open a browser window
-to give snapfu access to your Github organizations and to be able to create repositories in subsequent steps.
+```bash
+snapfu <command> <args> [--options]
+```
+
+## Commands
+
+### `init` - Create a new snap project
+Creates a new snap project (optional directory)
+
+```bash
+snapfu init <directory>
+```
+
+This command will:
+- Download scaffolding files
+- Create and initialize a repository in the Github organization you selected
+- Populate a Github secret with the provided `secretKey`
+
+You will need your `siteId` and `secretKey` from the SMC before you run this command.
```

Comment:
^ this is partially true of the `secretKey`.

Reply:
removed this too